src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -658,10 +669,23 @@ private PaymentProvider buildPaymentProvider(final AccountData account, @Nullabl
         paymentProvider.setCountryIsoCode(paymentProviderCountryIsoCode);
         paymentProvider.setPaymentType(paymentProviderPaymentType);
         paymentProvider.setRecurringType(paymentProviderRecurringType);
+        paymentProvider.setThreeDThreshold(parseThreeDThreshold(threeDThreshold));
```

Comment:
@arnelandwehr As far as I remember, the `Integer` was due to the column in `std_data` being `INTEGER`
```
pp_3ds_threshold INTEGER, -- DEFAULT NULL means there is no 3d secure needed, any other value is the threshold in MINOR UNITS
```
Since this `PaymentProvider` isn't directly bound to a DB anymore, we might as well use `Long` here for all setters & getters.